runtime/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Version 12.1000.0 - Mainnet
+  - Disabled channel payouts proposal
```

Comment:
Should we add `Renamed Gateway working group to App`?

Reply:
well spotted

Reply:
Fixed, and also updated from master (fix for graphql-server dependency on processor)